dts/arm/renesas/ra/ra2/ra2xx.dtsi

```diff
@@ -0,0 +1,247 @@
+			status = "disabled";
+		};
+
+		ioport6: gpio@400400c0 {
+			compatible = "renesas,ra8-gpio";
+			reg = <0x400400c0 0x20>;
+			port = <6>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		ioport7: gpio@400400e0 {
+			compatible = "renesas,ra8-gpio";
+			reg = <0x400400e0 0x20>;
+			port = <7>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		ioport8: gpio@40040100 {
+			compatible = "renesas,ra8-gpio";
+			reg = <0x40040100 0x20>;
+			port = <8>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		ioport9: gpio@40040120 {
+			compatible = "renesas,ra8-gpio";
+			reg = <0x40040120 0x20>;
+			port = <9>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <16>;
+			status = "disabled";
+		};
+
+		sci0: sci@40070000 {
+			compatible = "renesas,ra-sci";
+			interrupts = <4 1>, <5 1>, <6 1>, <7 1>;
+			interrupt-names = "rxi", "txi", "tei", "eri";
+			reg = <0x40070000 0x20>;
+			clocks = <&pclkb>;
+			status = "disabled";
+			uart {
+				compatible = "renesas,ra8-uart-sci";
+				channel = <0>;
+				status = "disabled";
+			};
+		};
+
+		sci1: sci@40070020 {
+			compatible = "renesas,ra-sci";
+			interrupts = <8 1>, <9 1>, <10 1>, <11 1>;
+			interrupt-names = "rxi", "txi", "tei", "eri";
+			reg = <0x40070020 0x20>;
+			clocks = <&pclkb>;
+			status = "disabled";
+			uart {
+				compatible = "renesas,ra8-uart-sci";
+				channel = <1>;
+				status = "disabled";
+			};
+		};
+
+		sci2: sci@40070040 {
+			compatible = "renesas,ra-sci";
+			interrupts = <12 1>, <13 1>, <14 1>, <15 1>;
+			interrupt-names = "rxi", "txi", "tei", "eri";
+			reg = <0x40070040 0x20>;
+			clocks = <&pclkb>;
+			status = "disabled";
+			uart {
+				compatible = "renesas,ra8-uart-sci";
+				channel = <2>;
+				status = "disabled";
+			};
+		};
+
+		sci3: sci@40070060 {
+			compatible = "renesas,ra-sci";
+			interrupts = <16 1>, <17 1>, <18 1>, <19 1>;
+			interrupt-names = "rxi", "txi", "tei", "eri";
+			reg = <0x40070060 0x20>;
+			clocks = <&pclkb>;
+			status = "disabled";
+			uart {
+				compatible = "renesas,ra8-uart-sci";
+				channel = <3>;
+				status = "disabled";
+			};
+		};
+
+		sci9: sci@40070120 {
+			compatible = "renesas,ra-sci";
+			interrupts = <24 1>, <25 1>, <26 1>, <27 1>;
+			interrupt-names = "rxi", "txi", "tei", "eri";
+			reg = <0x40070120 0x20>;
+			clocks = <&pclkb>;
+			status = "disabled";
+			uart {
+				compatible = "renesas,ra8-uart-sci";
+				channel = <9>;
+				status = "disabled";
+			};
+		};
+
+		flash-controller@407fe000 {
+			reg = <0x407fe000 0x1000>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+		};
+
+		id_code: idcode@1010018 {
+			compatible = "zephyr,memory-region";
+			reg = <0x01010018 0x20>;
+			zephyr,memory-region = "ID_CODE";
+			status = "okay";
+		};
+
+		option_setting_ofs: option_setting_ofs@0 {
+			compatible = "zephyr,memory-region";
+			reg = <0x00000000 0x18>;
```

Comment:
Do you refer to anywhere for this address value? As my understanding, it should be the address of OFS0 register as memtion in '7.2.1 Option Function Select Register 0 (OFS0) - RA2A1 MCU group HWM'

Reply:
FSP configures the options section.
So, this code can be removed, and I did so.

It may be necessary to accommodate additional settings, but this would require major changes, so we will consider this separately if necessary.